testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/BaseTestDeltaLakeMinioReads.java

```diff
@@ -72,20 +72,14 @@ public void tearDown()
     @Test(groups = {DELTA_LAKE_MINIO, PROFILE_SPECIFIC_TESTS})
     public void testReadRegionTable()
     {
-        onTrino().executeQuery(format(
-                "CREATE TABLE IF NOT EXISTS delta.default.\"%1$s\" (" +
-                        "  regionkey bigint, " +
-                        "  name varchar, " +
-                        "  comment varchar) " +
-                        "WITH (location = 's3://%2$s/%1$s')",
-                tableName,
-                BUCKET_NAME));
+        onTrino().executeQuery(format("CALL delta.system.register_table('default', '%1$s', 's3://%2$s/%1$s')", tableName, BUCKET_NAME));
 
         assertThat(onTrino().executeQuery(
                 format("SELECT count(*) FROM delta.default.\"%s\"", tableName)))
                 .containsOnly(row(5L));
 
         assertNotificationsCount(NOTIFICATIONS_TABLE, OBJECT_ACCESSED_HEAD, tableName + "/_delta_log/00000000000000000000.json", 0);
         assertNotificationsCount(NOTIFICATIONS_TABLE, OBJECT_ACCESSED_GET, tableName + "/_delta_log/00000000000000000000.json", 1);
+        onTrino().executeQuery(format("DROP TABLE delta.default.\"%s\"", tableName));
```

Comment:
Dropping the table, although necessary, seems unrelated to this commit. Could you please do it in a separate commit?

Reply:
Is it okay to have a separate PR to address this?

Reply:
FYI https://github.com/trinodb/trino/pull/14997